FAQ.md

```diff
@@ -91,7 +91,7 @@ This is set via the MCO’s config, not the Machine-api. The config a Machine re
 The user-data is generated by the installer.  The Machine-api is just a consumer of the user-data and simply passes the data to the cloud.  Refer to documentation for the Machine-config-operator (not part of the Machine-api) for creating new Machine pools and user-data.
 
 ## Adding Annotations and Labels to Nodes via Machines
-Annotations and Labels are only added to Nodes via Machines at creation time.  You can add arbitrary Annotations and Labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent Annotations and Labels on the Machine object (or in a MachineSet’s template) will have no effect on the Node.
+Annotations and Labels are added to Nodes via Machines whenever the machine or its node is updated. You can add arbitrary Annotations and Labels which are applied to Nodes immediately after they join the cluster. Removing Annotations or Labels from Machine won't remove them from its node. If you want to add an Annotation or Label that you will later remove, consider adding it directly to the node object. If you add it to Machine, you will need to remove it from the machine first to prevent it from being reapplied, and then remove it from the node object.
```

Comment:
```suggestion
Annotations and Labels are added to Nodes via Machines whenever the machine or its node is updated. You can add arbitrary Annotations and Labels which are applied to Nodes immediately after they join the cluster. Removing Annotations or Labels from a Machine won't remove them from its Node. If you want to add an Annotation or Label that you will later remove, consider adding it directly to the Node object. If you an Annotation or Label to a Machine, and later decide to remove it, you will need to remove it from the Machine first to prevent it from being reapplied, and then remove it from the Node object.
```